core/src/main/scala/zio/config/ConfigDescriptor.scala

```diff
@@ -80,6 +80,22 @@ sealed trait ConfigDescriptor[K, V, A] { self =>
   final def xmap[B](to: A => B, from: B => A): ConfigDescriptor[K, V, B] =
     self.xmapEither(a => Right(to(a)), b => Right(from(b)))
 
+  def mapKey(f: K => K): ConfigDescriptor[K, V, A] = {
```

Comment:
I'd make sure all methods are alphabetical in here, following the preceding conventions:

1. Operators before named methods.
2. Private / protected methods pushed to the very end.

Reply:
Done